index.html

```diff
@@ -442,7 +442,7 @@ <h3 class="bannerbox__title">I can offer a ride</h3>
                 <div class="bannerbox" id="story">
                     <h3 class="bannerbox__title">Our Story</h3>
                     <div class="bannerbox__content">
-                        <p>In many states, new voting rules have made it more difficult for people to get to polling stations. These rules particularly affect People of Color, People with Disabilities, Young People, Elderly People, and Women.Carpool Vote is a tool to fight back against an unfair political system in the 2016 election.</p>
+                        <p>In many states, new voting rules have made it more difficult for people to get to polling stations. These rules particularly affect People of Color, People with Disabilities, Young People, Elderly People, and Women. Carpool Vote is a tool to fight back against an unfair political system in the 2016 election.</p>
```

Comment:
Great catch 👍 :). While you're at it, can we make the minority groups lowercase, so that it reads as follows?
> people of color, people with disabilities, young people, elderly people, and women
These aren't proper nouns, so they shouldn't be capitalised. I fixed this in a previous version but it's recently been overwritten. CC @sasjkia.